en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -20,87 +20,98 @@ If there's indication that a vulnerability is actively exploited in the wild or
 
-## Handling security issues confidentially 
+## Handling security issues confidentially
 
 This process is relevant when a potential vulnerability is reported confidentially and the Silverstripe CMS development team is in a position to prepare a patch prior to the public disclosure of the vulnerability.
 
 This process is usually started once someone [reports a security issue](issues_and_bugs/#reporting-security-issues).
 
 ### When receiving a report
 
+[hint]
+Ensure the reporter is given a justification for all issues we conclude are not security vulnerabilities.
+[/hint]
+
 1. An automated response is sent back to the reporter to acknowledge receipt of their vulnerability report.
-2. Perform an initial assessment of the report.
-3. [Create a issue in our private security repository](https://github.com/silverstripe-security/security-issues/issues/new) unless the report is obviously invalid. e.g. SPAM.
-4. Clarify who picks up and owns the issue (assign in Github).
-  The owner can be separate from the developer resolving the issue,
-  their primary responsibility is to ensure the issue keeps moving through the process correctly.
-5. If encrypted information or attachments are provided, attach them to the private security issue.
-6. Reply to [security@silverstripe.org](mailto:security@silverstripe.org) ONLY (i.e. do not include the reporter in this reply) and provide a link to the private security issue. Keep most of the discussion on GitHub.
-7. Perform a criticality assessment to determine how severe the issue is and if it can be replicated.
+1. Perform an initial assessment of the report. If it is obviously spam or otherwise invalid, reply to the reporter thanking them for their efforts and explaining that we won't be handling this as a security issue.
+    - If what was raised is a valid bug but not a valid security report, advise them to raise an issue on GitHub.
+    - If the report was spam, see [spam](#spam) below.
+1. If the report is valid, [create a issue in our private security repository](https://github.com/silverstripe-security/security-issues/issues/new).
+    - Prefix the issue name with "[NO CVE REQUESTED YET]".
+    - If encrypted information or attachments are provided, attach them to the private security issue.
+    - Add the new issue to the "Triage" column on the [project board](https://github.com/silverstripe-security/security-issues/projects/1).
+1. Reply to [security@silverstripe.org](mailto:security@silverstripe.org) ONLY (i.e. do not include the reporter in this reply) and provide a link to the private security issue. Keep most of the discussion on GitHub.
+1. Clarify who picks up and owns the issue (assign in GitHub).
+    - The owner can be separate from the developer resolving the issue. Their primary responsibility is to ensure the issue keeps moving through the process correctly.
```

Comment:
```suggestion
    - The developer doing an initial assessment on the issue can be separate from the developer implementing a fix for the issue.
```

Reply:
Reworded a bit, since this suggestion assumed the person who picks up the issue was only doing the initial assessment.